inst/rmarkdown/templates/measles_outbreak/skeleton/skeleton.Rmd

```diff
@@ -627,8 +627,9 @@ linelist_cleaned <- linelist_cleaned %>%
 
 # change the order of levels of multiple categorical variables
 linelist_cleaned <- linelist_cleaned %>%
-  mutate_at(vars(starts_with("dehydration")), 
-            list(~factor(., levels = c("None", "Some", "Severe", "Unknown")))
+  mutate_at(vars(starts_with("dehydration")),
```

Comment:
hey neale just making sure - are there dehydration vars in measles? (thought it might only be in cholera, but i could be wrong). Just in case: that vars starts_with bit, searches for any variables in the dataset where that variable-name starts with "dehydration" (now that Ive typed it out sounds pretty obvious haha but just in case!!)